src/runtime/noop-proxy.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {StorageProxy} from './storage-proxy.js';
+import {HandleOld} from './handle.js';
+
+// Inject the function to allocate a NoOpStorageProxy, in order to remove
+// the dependency on StorageProxy from handle.js.
+HandleOld.noOpStorageAllocator = StorageProxy.newNoOpProxy;
```

Comment:
So this is going to be replaced by Handle (with StorageNG)? Just trying to understand the naming.

Reply:
No idea. I'm just moving a direct call with an assigned one. The only name I came up with was noOpStorageAllocator. The rest were already there and I didn't change them.

Reply:
We'll leave it to the storage folks then.